### DISCUSSION TITLE
Pull request template changes?

### DISCUSSION THREAD

**Discussion by ghost** [atom-ide-community/atom#414] [General] [answered]
The pull request are the same as atom/atom (upstream) repository. AFAIK, the atom community project is trying to distance from Atom. So what about `PULL_REQUEST_TEMPLATE.md`?

**Comment by meadowsys** [comment, marked as answer]
I think that the template being the same as upstream is not that big of an issue, as long as everything is still applicable. Although the copy/paste for pull request templates is not great. Github [supports having multiple pull request templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository), so the copy/paste could be replaced by those templates. Maybe as the templates are being migrated over we can check them and edit things as needed so they are more applicable for this fork

**Reply by ghost** [reply]
Maybe I could fork and PR then

**Reply by aminya** [reply]
These can change in 
https://github.com/atom-community/.github

**Reply by ghost** [reply]
> These can change in https://github.com/atom-community/.github

i'll make my pr later @aminya
